File: easy/swift/onboarding.swift
import Foundation

while true {
    let enemy1 = readLine()!
    let dist1 = Int(readLine()!)!
    let enemy2 = readLine()!
    let dist2 = Int(readLine()!)!

    if dist1 < dist2 {
        print(enemy1)
    } else {
        print(enemy2)
    }
}

File: easy/ocaml/onboarding.ml
let rec main () =
  let enemy1 = read_line () in
  let dist1 = int_of_string (read_line ()) in
  let enemy2 = read_line () in
  let dist2 = int_of_string (read_line ()) in

  if dist1 < dist2 then
    print_endline enemy1
  else
    print_endline enemy2;
  
  main ();;

let () = main ()
